### DISCUSSION TITLE
Coverage Test Failed

### DISCUSSION THREAD

**Discussion by terrylin8** [CS342/.github#63] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

My coverage test is failing and preventing me from completing the PR. When reran with error logs this is one of the errors: 
"Upload Coverage Report / Create and upload coverage report
The job was not started because recent account payments have failed or your spending limit needs to be increased. Please check the 'Billing & plans' section in your settings."
![Screenshot 2024-02-06 at 22 47 53](https://github.com/CS342/.github/assets/56202767/dfc66b9f-7b63-4089-9ba5-61e3623639af)

Please let me know if this is something I can resolve by the midterm presentation. Thank you! 

### Reproduction

Upload Coverage Report

### Expected behavior

Pass

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@twtl67 Sorry about this issue. We updated the settings and I re-triggered the workflow. It should work now 👍

**Reply by terrylin8** [reply, marked as answer]
It's working now, thanks Paul!
